store-and-retrieve-data/use-storagehub-sdk/create-a-bucket.md

```diff
@@ -1,4 +1,237 @@
 ---
 title: Create a Bucket
-description: Guide to creating a new storage bucket using StorageHub SDK.
+description: Guide on how to create a new storage bucket with the StorageHub SDK.
 ---
+
+# Create a Bucket
```

Comment:
I would love to see a concise intro before Prerequisites... e.g. this page will show you how to create a bucket programmatically using the Storage Hub SDK.